pandas/tests/sparse/test_combine_concat.py

```diff
@@ -4,31 +4,61 @@
 import numpy as np
 import pandas as pd
 import pandas.util.testing as tm
+from pandas.errors import PerformanceWarning
 import itertools
 
 
+class TestSparseArrayConcat(object):
+    @pytest.mark.parametrize('kind', ['integer', 'block'])
+    def test_basic(self, kind):
+        a = pd.SparseArray([1, 0, 0, 2], kind=kind)
+        b = pd.SparseArray([1, 0, 2, 2], kind=kind)
+
+        result = pd.SparseArray._concat_same_type([a, b])
```

Comment:
this let's me think, should we consider to expand `pd.concat` to also concatenating arrays? (not for this PR of course :-))

Reply:
I think https://github.com/pandas-dev/pandas/issues/22994 will cover that.